### PR TITLE
Bonne annee 2023

### PR DESCRIPTION
**Pull request by imrhiel**
Ajout du post de bonne année rédigé par Lovis_IX et sa traduction en anglais

### CODE REVIEW

File: content/post/bonne-annee-2023.en.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Happy new year 2023"
+date: 2023-01-14
+draft: false
+---
+
+Here we are, our planet just finished another revolution around its star and started a new one.
+
+In all our names, please allow me to wish you a good and happy new year 2023.
```
Comment (Porkepix) [inline content/post/bonne-annee-2023.en.md:9]: On everyone's/everybody's behalf instead?

File: content/post/bonne-annee-2023.en.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+From the outside, the year 2022 wasn't extremely active for the association.
+Not a lot of meetings and talks, because of the lack of volunteer time.
+
+- Some new members arrived and it is extremely positive for the association. They are still a bit shy during our monthly meetings, understandably, they are testing the waters; but I hope they will soon be active in all the domains Exodus Privacy is offering. There is so much to do;
+- The platform keeps going on its merry way;
+- Our infrastructure is evolving, slowly distancing itself from the first-ever servers we set up at the beginning of our adventure;
+- Our finances are healthy thanks to our treasurer;
+- We are trying to communicate more and better. It isn't really in our nature, maybe our new members will help us with this;
```
Comment (Porkepix) [inline content/post/bonne-annee-2023.en.md:18]: Could be "on that regard" or "on that topic', but to be honest I'm not proficient enough in English to know which is better and the current form is fine as is I think.

File: content/post/bonne-annee-2023.en.md (hunk 1 above)
Comment (imrhiel) [reply content/post/bonne-annee-2023.en.md]: Changed ("on behalf of all of us")

File: content/post/bonne-annee-2023.en.md (hunk 2 above)
Comment (imrhiel) [reply content/post/bonne-annee-2023.en.md:18]: I think both work

**Review by codeurimpulsif — APPROVED** [review]
Ok for me :)

**Pull request merged**